### DISCUSSION TITLE
From fresh Ubuntu to Frigate with Home Assistant integration

### DISCUSSION THREAD

**Discussion by simondrake** [blakeblackshear/frigate#4538] [Show and tell]
Went through the process of installing Frigate/Home Assistant on a fresh Ubuntu installation over the weekend. 

Documented the steps in case it helps anybody - https://www.simondrake.dev/blog/frigate-and-homeassistant

Edit: Update link after blog migration.

**Comment by blakeblackshear** [comment]
Nice guide. Thanks for sharing. Only comment would be to set detect fps to something lower than 30. That can be lower than the camera setting without consequence and you should still see good performance with 5 fps.

**Reply by simondrake** [reply]
Thanks @blakeblackshear, and thanks for making Frigate! 

I've updated the article to recommend 5 fps instead of 30.

If my camera supports 30, is there still a good reason to use 5?

**Reply by blakeblackshear** [reply]
Definitely. Each frame sent for detection will be processed by frigate. It should reduce CPU load of frigate processing by 6x going from 30 to 5. Most of the frames will be nearly identical at 30fps, so it's spending a lot of time processing the same image over and over.

**Reply by simondrake** [reply]
I see, thanks! I'll change my own settings while I'm at it then 😄

**Reply by simondrake** [reply]
@blakeblackshear (while we're already talking) - is there a way to make the birdseye view full screen?

I want to get a few small screens round the house (not sure which ones yet), to show the cameras in birdseye view.

**Reply by simondrake** [reply]
Obviously tell me to piss off and raise a new thread if you want :)

**Reply by NickM-27** [reply]
You can make it full screen by clicking on it. There's an open request to have a query parameter that does this automatically

**Reply by simondrake** [reply]
Well that serves me right for not checking before asking! 

Thanks :)

**Reply by simondrake** [reply]
> You can make it full screen by clicking on it. There's an open request to have a query parameter that does this automatically 

FYI - the tap to make full screen doesn't work on mobile (iOS / Safari). Not sure it's that's intentional?

**Reply by NickM-27** [reply]
I believe fullscreen from that element is not something that iOS supports

**Reply by simondrake** [reply]
> I believe fullscreen from that element is not something that iOS supports

Ah ok 👍 thanks!

**Comment by skunk-funk** [comment]
Was this guide taken down?

**Reply by simondrake** [reply]
I recently did a blog migration and the slugs changed as a result. I wasn't aware anybody was still using it, but I've edited to add the new URL which is https://www.simondrake.dev/blog/frigate-and-homeassistant.

**Reply by skunk-funk** [reply]
Excellent, thank you!
